src/components/bulk-email-tool/bulk-email-form/bulk-email-recipient/BulkEmailRecipient.jsx

```diff
@@ -69,6 +69,7 @@ export default function BulkEmailRecipient(props) {
             <Form.Checkbox
               key={cohort}
               value={`cohort:${cohort}`}
+              disabled={selectedGroups.find((group) => group === DEFAULT_GROUPS.ALL_LEARNERS)}
```

Comment:
Holy moly JS syntax makes my brain melt sometimes. I think this just says "Cohort checkboxes are disabled if `ALL_LEARNERS` is in the `selectedGroups`... list/collection?

Reply:
Yes and ALL_LEARNERS === "learners" so `selectedGroups` is just an array of strings